zerver/webhooks/updown/view.py

```diff
@@ -19,7 +19,7 @@ class UnsupportedUpdownEventType(JsonableError):
     pass
 
 def send_message_for_event(event, user_profile, client, stream):
-    # type: (Dict[str, Any], UserProfile, Client, str) -> None
+    # type: (Dict[str, Any], UserProfile, Client, str) -> Optional[HttpResponse]
```

Comment:
I think the bug here is actually that it should be doing `raise JsonableError()`, not `return json_error`